docs/content/installation/installation-with-manifests.md

```diff
@@ -101,6 +101,7 @@ Create a custom resource definition for `APPolicy`, `APLogConf` and `APUserSig`:
    $ kubectl apply -f common/crds/appprotect.f5.com_appolicies.yaml
    $ kubectl apply -f common/crds/appprotect.f5.com_apusersigs.yaml
    ```
+> **Note** This step can be skipped if you are using App Protect WAF module with policy bundles.
```

Comment:
This line still needs to be moved up to Line 96 so that it is presented to the user before hand. Right now it is placed below the `kunectl` commands and it is likely that the user run the commands and then read the note after.